plugins/scaffolder/src/components/OngoingTask/OngoingTask.tsx

```diff
@@ -118,25 +134,28 @@ export const OngoingTask = (props: {
       search: `?${qs.stringify({ formData: JSON.stringify(formData) })}`,
     });
   }, [
+    analytics,
     navigate,
     taskStream.task?.spec.parameters,
     taskStream.task?.spec.templateInfo?.entity?.metadata,
+    templateRef,
     templateRouteRef,
   ]);
 
   const [{ status: cancelStatus }, { execute: triggerCancel }] = useAsync(
     async () => {
       if (taskId) {
+        analytics.captureEvent(
+          'click',
```

Comment:
I wonder if it makes sense for us to remove the `Task has been canceled` from the subject, and actually just use more semantic events. Like `captureEvent('cancelled', templateRef)` instead?

Reply:
changed it